aries-site/package.json

```diff
@@ -10,6 +10,9 @@
     "grommet": "https://github.com/grommet/grommet/tarball/stable",
     "grommet-icons": "https://github.com/grommet/grommet-icons/tarball/stable",
     "grommet-theme-hpe": "https://github.com/grommet/grommet-theme-hpe/tarball/NEXT-stable",
+    "imagemin-svgo": "^7.1.0",
+    "next-compose-plugins": "^2.2.0",
+    "next-optimized-images": "^2.5.7",
```

Comment:
Can you explain at the PR description why you added those packages and how they help to achieve the PR goal?